sdk/storage/azure-storage-file-datalake/GEN1_GEN2_MAPPING.md

```diff
@@ -0,0 +1,163 @@
+<h1>Mapping from ADLS Gen1 API -> ADLS Gen2 API</h1>
+<table style="background:white">
+<thead>
+<tr>
+<th>ADLS Gen1 API</th>
+<th>Note for Gen1 API</th>
+<th>ADLS Gen2 API</th>
+<th>Note for API Mapping</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>checkAccess</td>
+<td>Checks whether the calling user has the required permissions for the file/directory.</td>
+<td>N/A</td>
+<td></td>
+</tr>
+<tr>
+<td>checkExists</td>
+<td>Check whether a file or directory exists.</td>
+<td>N/A</td>
+<td>Use Gen2 API methods <strong>createFileWithResponse</strong> and <strong>createDirectoryWithResponse</strong> and set <strong>BlobRequestConditions.setIfNoneMatch(true)</strong></td>
```

Comment:
I think this needs to be setIfNoneMatch("*"). And didn't I also see you create an issue for adding an exists method()? That would be a better mapping

Reply:
You're right, thanks for that catch. And I created the two issues for exists and no overwrite default, we'll have to update the doc once those features exist

Reply:
fixed